plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -104,10 +104,22 @@ object ScalafmtPlugin extends AutoPlugin {
     val repositories = resolvers.collect {
       case r: MavenRepository => r.root
     }
-    val scalafmtInstance =
+    val scalafmtSession =
       globalInstance
         .withReporter(reporter)
         .withMavenRepositories(repositories: _*)
+        .withRespectProjectFilters(true) match {
```

Comment:
this line wasn't there before. probably a good change. could it potentially cause some complaints?

Reply:
Looks like it's [on-by-default](https://github.com/scalameta/scalafmt/blob/ebeb78a10f75ffe3be5d0aa7d965a874a940a64c/scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtDynamic.scala#L38). Just explicitly declare this setting because sbt plugin hasn't other inputs than filters specified in config.